manage-data/lifecycle/index-lifecycle-management/policy-apply.md

```diff
@@ -0,0 +1,72 @@
+---
+navigation_title: Apply a policy to an index
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# Apply a lifecycle policy to an index [apply-policy-manually]
+
+When you create new {{es}} index you can use an index template to apply the lifecycle policy by which the index will be managed. This process is described in [Configure a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md).
+
+You can also manually apply a lifecycle policy to an existing index, as described here. If an index is currently managed by an ILM policy you must first remove that policy before applying a new one. Refer to [Switch to a different lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/policy-updates.md#switch-lifecycle-policies) for details.
+
+You can do this procedure in {{kib}} or using the {{es}} API.
+
+::::{warning}
+Do not manually apply a policy that uses the rollover action to an index which has not yet rolled over. Otherwise, the policy may not be carried forward when the rollover action creates a new index.
```

Comment:
🙈 Sorry, why not as long as the template is setup to catch?

Reply:
This is the note as it appears in the current docs [here](https://www.elastic.co/docs/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy#apply-policy-manually):


<img width="667" alt="rollover-action-note" src="https://github.com/user-attachments/assets/f07f3bb5-59f8-47ac-bb48-11fa5fa66f36" />

---

Please let me know if you think we should rephrase it somehow, or otherwise I can just remove it.